popgen/dunn.go

```diff
@@ -58,15 +53,11 @@ func FindMaxIntra(subFa []*fasta.Fasta, g *Group, b *bed.Bed) int {
 //FindMinInter is a helper function of Dunn that calculates the minimum pairwise sequence distance between two sequences of a multiFa alignment that are part of different groups.
 func FindMinInter(g []*Group, subFa []*fasta.Fasta) int {
 	var answer int = math.MaxInt64
```

Comment:
This would be a use of the new numbers.MaxInt
Is max int what we would want for the return value when empty groups are given?  0 or a panic instead? @rimangan